experiments/misc/ilamb_setup.jl

```diff
@@ -0,0 +1,146 @@
+import Dates
```

Comment:
maybe we can rename the folder `experiments/misc/` to `experiments/ilamb/` since everything in that folder is for ilamb